tests/phpunit/tests/post/output.php

```diff
@@ -22,15 +22,21 @@ function tearDown() {
 
 	function _shortcode_dumptag( $atts ) {
 		$out = '';
-		foreach ($atts as $k=>$v)
+		foreach ( $atts as $k => $v ) {
 			$out .= "$k = $v\n";
+		}
 		return $out;
 	}
 
 	function _shortcode_paragraph( $atts, $content ) {
-		extract(shortcode_atts(array(
-			'class' => 'graf',
-		), $atts));
+		extract(
```

Comment:
Agreed and I expect this will get picked up in a future rule addition.

Reply:
The rule here is:
WordPress.PHP.DontExtract.extract_extract
It is currently a warning rather than an error but this will need picking up and fixing in a future PR.

Reply:
As per review meeting, to be addressed in future PR.